test/test_utils/ec2.py

```diff
@@ -536,6 +544,35 @@ def execute_ec2_training_performance_test(
     )
 
 
+def execute_ec2_habana_training_performance_test(
+    connection, ecr_uri, test_cmd, region=DEFAULT_REGION, data_source="", cards_num=None):
+    docker_cmd = "docker"
+    container_test_local_dir = os.path.join("$HOME", "container_tests")
+
+    timestamp = time.strftime("%Y-%m-%d-%H-%M-%S")
+    log_name = f"{data_source}_results_{os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')}_{timestamp}.txt"
+    synapseai_version = get_synapseai_version_from_tag(ecr_uri)
+    # Make sure we are logged into ECR so we can pull the image
+    connection.run(f"$(aws ecr get-login --no-include-email --region {region})", hide=True)
+
+    connection.run(f"{docker_cmd} pull -q {ecr_uri}")
```

Comment:
Is there a special purpose to preload the container?

Reply:
No, nothing special. The function definition is referred from `execute_ec2_training_performance_test`. Optionally we can skip the pull. Will get this addressed in separate PR